bee-network/bee-autopeering/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bee-autopeering"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
@thibault-martinez I'm not sure if it applies here, but I thought we were always bumping minor?

Reply:
For consistency it would be best to only bump minor yeah. But since the previous versions of this weren't published this shouldn't be an issue.

Reply:
Isn't this PR literally patching `bee-autopeering`, so bumbing the patch version should apply. Bumping the minor is for backwards compatibly feature additions. Major is for backwards incompatible changes. At least this is how I understood semantic versioning. But maybe I forgot or didn't read a certain discussion.

Reply:
Cargo semver is different from the usual semver

Reply:
It's preferred to bump the minor and not the patch in every case for crates < 1.0.0. Pretty much the whole ecosystem is following this rule.

Reply:
Done